peekaboo/toolbox/cortex.py

```diff
@@ -117,13 +169,80 @@ def get_submit_parameters(self, sample, submit_original_filename=False):
 
 class FileInfoAnalyzerReport(CortexAnalyzerReport):
     """ Represents a Cortex FileInfo_7_0 analysis JSON report. """
-    def __init__(self, report):
-        self.report = report
+
+    report_schema = schema.Schema({
+        "summary": {
+            "taxonomies": [schema.Schema({
+                "level": schema.Or("info", "malicious", "safe"),
+                "namespace": "FileInfo",
+            #    "predicate": str,
+            #    "value": str
+            }, ignore_extra_keys=True)]
+        },
+        "full": {
+            "results": [
+            {
+                "submodule_name": "Basic properties",
+                "results": [
+                    {
+                        "submodule_section_header": "Hashes",
+                        "submodule_section_content": {
+                            "md5": schema.Regex(r'^[0-9a-z]{32}$'),
+                            "sha1": schema.Regex(r'^[0-9a-z]{40}$'),
+                            "sha256": schema.Regex(r'^[0-9a-z]{64}$'),
+                            "ssdeep": schema.Regex(r'^[0-9A-Za-z:]*$'),
+                        }
+                    },
+                    {
+                        "submodule_section_header": schema.And(str, lambda s: s != "Hashes"),
```

Comment:
Should we have a comment here what this does so we remember in two weeks' time?

Reply:
Definitely.